modules/distr-tracing-tempo-deploy-default.adoc

```diff
@@ -0,0 +1,132 @@
+====
+
+.Prerequisites
+
+* The {TempoName} Operator has been installed.
+* You have reviewed the instructions for how to customize the deployment.
+* You have access to the cluster as a user with the `cluster-admin` role.
+* Supported Object Store (AWS S3, Google Cloud Storage, Azure, Minio, OpenShift Data Foundation)
+
+.Procedure
+
+. Log in to the {product-title} web console as a user with the `cluster-admin` role.
+
+. Create a new project, for example `tracing-system`.
++
+[NOTE]
+====
+If you are installing as part of Service Mesh, the {DTShortName} resources must be installed in the same namespace as the `ServiceMeshControlPlane` resource, for example `istio-system`.
+====
++
+.. Navigate to *Home* -> *Projects*.
+
+.. Click *Create Project*.
+
+.. Enter `tracing-system` in the *Name* field.
+
+.. Click *Create*.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. If necessary, select `tracing-system` from the *Project* menu. You may have to wait a few moments for the Operators to be copied to the new project.
+
+. Click the {TempoName} Operator. On the *Details* tab, under *Provided APIs*, the Operator provides a single link.
+
+. Under *Tempo*, click *Create Instance*.
+
+. On the *Create Tempo* page, to install using the defaults, click *Create* to create the {TempoShortName} instance.
+
+. On the *Tempos* page, click the name of the {TempoShortName} instance, for example, `example`.
+
+. On the *Tempo Details* page, click the *Resources* tab. Wait until the pod has a status of "Running" before continuing.
+
+
+[id="distr-tracing-deploy-default-cli_{context}"]
+== Deploying the {DTShortName} default strategy from the CLI
+
+Follow this procedure to create an instance of {TempoShortName} from the command line.
+
+.Prerequisites
+
+* The {TempoName} Operator has been installed and verified.
+* You have reviewed the instructions for how to customize the deployment.
+* You have access to the OpenShift CLI (`oc`) that matches your {product-title} version.
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Log in to the {product-title} CLI as a user with the `cluster-admin` role.
++
+[source,terminal]
+----
+$ oc login --username=<NAMEOFUSER> https://<HOSTNAME>:8443
+----
+
+. Create a new project named `tracing-system`.
++
+[source,terminal]
+----
+$ oc new-project tracing-system
+----
+
+. Create a Secret YAML file to specify the secret for your object storage bucket. For instance, if you are using AWS S3, create a file that uses the `access_key_id` and `access_key_secret` fields to specify your AWS credentials and bucketnames, endpoint and region to define the object storage location. For example:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: minio-secret
+stringData:
+  endpoint: http://minio.minio.svc:9000
+  bucket: tempo
+  access_key_id: tempo
+  access_key_secret: supersecret
```

Comment:
Would it be possible to add some more realistic example of the key secret value?